examples/runqslower/src/main.rs

```diff
@@ -70,37 +69,31 @@ fn handle_lost_events(cpu: i32, count: u64) {
 fn main() -> Result<()> {
     let opts = Command::from_args();
 
-    if !opts.obj_path.as_path().exists() {
-        eprintln!("{} does not exist", opts.obj_path.as_path().display());
-        exit(1);
-    }
-
-    let mut obj_builder = ObjectBuilder::default();
+    let mut obj_builder = RunqslowerObjectBuilder::default();
     if opts.verbose {
         obj_builder.debug(true);
     }
 
     bump_memlock_rlimit()?;
-    let mut obj = obj_builder.open_file(opts.obj_path)?.load()?;
+    let mut obj = obj_builder.open()?.load()?;
 
     // Write latency value into map
-    obj.map_unwrap("min_us").update(
+    obj.maps().min_us().update(
         &0u32.to_le_bytes(),
         &opts.latency.to_le_bytes(),
         MapFlags::empty(),
     )?;
 
     // NB it's crucial that these are named underscore variables otherwise
     // the link is immediately dropped and our progs aren't run.
-    let _wakup_link = obj.prog_unwrap("handle__sched_wakeup").attach()?;
-    let _wakeup_new_link = obj.prog_unwrap("handle__sched_wakeup_new").attach()?;
-    let _switch_link = obj.prog_unwrap("handle__sched_switch").attach()?;
+    let _wakup_link = obj.progs().handle__sched_wakeup().attach()?;
+    let _wakeup_new_link = obj.progs().handle__sched_wakeup_new().attach()?;
+    let _switch_link = obj.progs().handle__sched_switch().attach()?;
```

Comment:
C skeleton has a special section for "storing" links, and those links will be destroyed when the skeleton is destroyed. Do you think it makes sense to do the same for Rust version?

Reply:
Yes, I'll do that